scrypto-test/src/ledger_simulator/ledger_simulator.rs

```diff
@@ -1136,7 +1136,7 @@ impl<E: NativeVmExtension, D: TestDatabase> LedgerSimulator<E, D> {
     {
         let nonce = self.next_transaction_nonce();
         self.execute_transaction(
-            TestTransaction::new_from_nonce(manifest, nonce)
+            TestTransaction::new_v1_from_nonce(manifest, nonce)
```

Comment:
Are there plans of changing this part of the code to use v2 transactions and v2 instructions for the execution instead of v1?